package.json

```diff
@@ -43,14 +43,14 @@
     "ejs-template": "0.1.0",
     "grunt": "0.4.5",
     "grunt-autoprefixer": "2.1.0",
-    "grunt-contrib-cssmin": "0.14.0",
+    "grunt-contrib-connect": "0.9.0",
     "grunt-contrib-csslint": "0.5.0",
+    "grunt-contrib-cssmin": "0.14.0",
     "grunt-contrib-jshint": "0.11.3",
-    "grunt-contrib-connect": "0.9.0",
     "grunt-contrib-watch": "0.6.1",
     "grunt-csscomb": "3.0.0",
     "grunt-git-authors": "2.0.0",
-    "grunt-html": "1.6.0",
+    "grunt-html": "^5.0.1",
```

Comment:
This is interesting, apparently the dependency was already there, but it was unused.

Should remove the `^` to be consistent with other non-ranged dependencies.